scrapers/Hucows.yml

```diff
@@ -19,12 +19,12 @@ xPathScrapers:
               - regex: ^\s+
                 with:
           - parseDate: 2 Jan 2006
-      Image: //center/a[1]/img[@class="lightboxhover"]/@src
+      Image: //div[@class="imghoverclass postfeat post-single-img"]//img//@src
```

Comment:
shouldnt it be `//img/@src` ?
we could also future proof it a bit by using
`//div[contains(@class,"post-single-img")]//img/@src` assuming `post-single-img` will last longer than the rest of the class name

Reply:
fixed